src/components/navigation/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import menuLinks from '../../commons/menu';
+
+function Header() {
+  return (
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use `Arrow function` on your `Header` component for best practices. I suggest you apply this approach in all your components